fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/automatic/PrefAutomaticTranslationItem.kt

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.preferences.automatic
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.mozilla.fenix.R
+
+/**
+ * AutomaticTranslationItem that will appear on Automatic Translation screen.
+ *
+ * @property displayName The text that will appear in the list.
+ * @property prefAutomaticTranslationOption The option that the user selected.
+ */
+@Parcelize
+data class PrefAutomaticTranslationItem(
+    val displayName: String,
+    val prefAutomaticTranslationOption: PrefAutomaticTranslationOption,
+) : Parcelable
+
+/**
+ * AutomaticTranslationOption for a language.
+ *
+ * @property titleId The string id title of the option.
+ * @property summaryId The string id summary of the option.
+ */
+@Parcelize
+sealed class PrefAutomaticTranslationOption(
+    open val titleId: Int,
+    open val summaryId: List<Int>,
+) : Parcelable {
+
+    /**
+     * The app will offer to translate sites in the selected language.
+     */
+    data class OfferToTranslate(
+        override val titleId: Int = R.string.pref_automatic_translation_option_offer_to_translate_title,
+        override val summaryId: List<Int> = listOf(
+            R.string.pref_automatic_translation_option_offer_to_translate_summary,
+            R.string.app_name,
```

Comment:
I would adjust this string and the others using `R.string.app_name` because the copy is:

`Firefox will offer to translate sites in this language.`

and using `app_name` on my build produces:

`Firefox Fenix will offer to translate sites in this language.`

I did a check through the strings.xml and it seems "Firefox" is used in strings. (I didn't see a freestanding "Firefox" for us to substitute in, but maybe there is somewhere?)

Reply:
Done